### PR TITLE
fix: gql docs for update asset + rename minLpStake to quantum in proposals

### LINKED ISSUE #5904: [graphql] Fix documentation for udpateAsset
https://github.com/vegaprotocol/vega/blob/277b3e9318ccc772df34fd49fe039d8283993f8b/datanode/gateway/graphql/schema.graphql#L2959

#5281 added updateAsset to graphql, but the graphql documentation is copy and pasted incorrectly

# Task
- [ ] Make documentation correct

### PR DESCRIPTION
**Pull request by jeremyletang**
close #5904

### CODE REVIEW

File: datanode/gateway/graphql/schema.graphql [hunk 1]
```diff
@@ -2949,23 +2949,23 @@ type NewAsset {
...
-  minLpStake: String!
+  "The minimum economically meaningful amount in the asset"
+  quantum: String!
 
   "The source of the new asset"
   source: AssetSource!
 }
 
-"A new asset proposal change"
+"A a proposal to update an asset details"
```
Comment (edd) [inline datanode/gateway/graphql/schema.graphql:2959]:
```suggestion
"A proposal to update an asset details"
```

**Review by edd — DISMISSED** [review]

**Review by EVODelavega — DISMISSED** [review]

**Review by EVODelavega — DISMISSED** [review]

**Review by EVODelavega — DISMISSED** [review]

File: datanode/gateway/graphql/schema.graphql [hunk 2]
```diff
@@ -2949,23 +2949,23 @@ type NewAsset {
...
-  minLpStake: String!
+  "The minimum economically meaningful amount in the asset"
+  quantum: String!
 
   "The source of the new asset"
   source: AssetSource!
 }
 
-"A new asset proposal change"
+"A proposal to update an asset details"
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:2959]:
```suggestion
"A proposal to update an asset's details"
```

File: datanode/gateway/graphql/schema.graphql [hunk 3]
```diff
@@ -2949,23 +2949,23 @@ type NewAsset {
   "The precision of the asset"
   decimals: Int!
 
-  "The minimum stake to become a liquidity provider for any market using this asset for settlement"
-  minLpStake: String!
+  "The minimum economically meaningful amount in the asset"
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:2952]:
```suggestion
  "The minimum economically meaningful amount of this specific asset"
```

File: datanode/gateway/graphql/schema.graphql [hunk 4]
```diff
@@ -2949,23 +2949,23 @@ type NewAsset {
...
 type UpdateAsset {
   "The full name of the asset (e.g: Great British Pound)"
   name: String!
 
   "The symbol of the asset (e.g: GBP)"
   symbol: String!
 
-  "The minimum stake to become a liquidity provider for any market using this asset for settlement"
-  minLpStake: String!
+  "The minimum economically meaningful amount in the asset"
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:2967]:
```suggestion
  "The minimum economically meaningful amount of this specific asset"
```

**Review by candida-d — COMMENTED** [review]
non-blocking suggestions for rewording the descriptions

**Review by EVODelavega — APPROVED** [review]

**Review by mattrussell36 — APPROVED** [review]

**Pull request merged**